README.md

```diff
@@ -224,10 +224,122 @@ __Objective-C__
+the IDFA.
+
+__Swift__
+```swift
+import AppTrackingTransparency
+import AdSupport
+
+func useIDFA(completionHandler: @escaping (Bool, String) -> Void) {
+  if #available(iOS 14, *) { // After iOS 14, we request the IDFA to the AppTrackingTransparency framework
+    ATTrackingManager.requestTrackingAuthorization { status in
+      let isTrackingEnabled = status == .authorized
+      var idfa: String = ""
+      if isTrackingEnabled {
+        idfa = ASIdentifierManager.shared().advertisingIdentifier.uuidString
+      }
+      completionHandler(isTrackingEnabled, idfa)
+    }
+  } else { // On older devices, we can access the IDFA directly
+    let isTrackingEnabled = ASIdentifierManager.shared().isAdvertisingTrackingEnabled
+    var idfa: String = ""
+
+    if trackingIsAllowed {
+      idfa = ASIdentifierManager.shared().advertisingIdentifier.uuidString
+    }
+    completionHandler(isTrackingEnabled, idfa)
+  }
+}
+```
+
+__Objective-C__
+```objectivec
+#import <AdSupport/ASIdentifierManager.h>
+@import AppTrackingTransparency;
+
+- (void)useIDFA:(void (^)(bool, NSString *))completionHandler {
+  if (@available(iOS 14, *)) { // After iOS 14, we request the IDFA to the AppTrackingTransparency framework
+    [ATTrackingManager requestTrackingAuthorizationWithCompletionHandler:^(
+                           ATTrackingManagerAuthorizationStatus status) {
+      bool isTrackingEnabled = status == ATTrackingManagerAuthorizationStatusAuthorized;
+      NSString *idfa = nil;
+      if (isTrackingEnabled) {
+        idfa = [[[ASIdentifierManager sharedManager] advertisingIdentifier] UUIDString];
+      }
+      completionHandler(isTrackingEnabled, idfa);
+    }];
+  } else { // On older devices, we can access the IDFA directly
+    bool isTrackingEnabled = [[ASIdentifierManager sharedManager] isAdvertisingTrackingEnabled];
+    NSString *idfa = nil;
+    if (isTrackingEnabled) {
+      idfa = [[[ASIdentifierManager sharedManager] advertisingIdentifier] UUIDString];
+    }
+    completionHandler(isTrackingEnabled, idfa);
+  }
+}
+```
+
+Once we established a method of requesting permission for using the IDFA, we just have to call it and provide a completion handler where we are creating the event.
+
+__Swift__
+```swift
+useIDFA(completionHandler: { (isTrackingEnabled, idfa) in
+  if isTrackingEnabled {
+    let userId = VSDKUserId(id: idfa, type: "idfa")
+    // ...
+  }
+})
+```
+
+__Objective-C__
+```objectivec
+[self useIDFA:^(bool isTrackingEnabled, NSString *idfa) {
+  if (isTrackingEnabled) {
+    VSDKUserId *userId = [[VSDKUserId alloc] initWithId:idfa type:@"idfa"];
+    // ...
+  }
+}];
+```
+
+More information on how to retrieve the user's IDFA can be found in Apple's documentation of the [App Tracking Transparency framework](https://developer.apple.com/documentation/apptrackingtransparency?language=objc).
```

Comment:
I think the pointer to the App Tracking Transparency framework should be put at the beginning of this section.

Reply:
Done in ce9fb2d